.github/workflows/build_wheel.yml

```diff
@@ -0,0 +1,281 @@
+name: Build Wheels
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*"
+  pull_request:
+
+jobs:
+  build_wheels_linux:
+    name: Build wheels on Linux
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ">=3.10.0"
+
+      - name: Install cibuildwheel
+        run: pip install cibuildwheel==2.20.0
+
+      - name: Build wheels
+        env:
+          CIBW_BUILD: "cp310-manylinux_x86_64 cp311-manylinux_x86_64 cp312-manylinux_x86_64"
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux_2_28
+          CIBW_BUILD_VERBOSITY: "3"
+          CIBW_BEFORE_ALL: |
+            yum install -y wget
+            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+            bash miniconda.sh -b -p /root/miniconda
+            export PATH="/root/miniconda/bin:$PATH"
+            conda config --set always_yes yes --set changeps1 no
+            conda update -q conda
+            conda info -a
+            conda config --add channels conda-forge
+            conda config --set channel_priority strict
+            conda create -n build_env python=3.10
+            source activate build_env
+            conda install -y sleef
+          CIBW_ENVIRONMENT: >
+            LD_LIBRARY_PATH="/root/miniconda/envs/build_env/lib:$LD_LIBRARY_PATH"
+            LIBRARY_PATH="/root/miniconda/envs/build_env/lib:$LIBRARY_PATH"
+            CFLAGS="-I/root/miniconda/envs/build_env/include $CFLAGS"
+            CXXFLAGS="-I/root/miniconda/envs/build_env/include $CXXFLAGS"
+            LDFLAGS="-L/root/miniconda/envs/build_env/lib $LDFLAGS"
+            SLEEF_PATH="/root/miniconda/envs/build_env"
+          CIBW_REPAIR_WHEEL_COMMAND: >
+            auditwheel repair -w {dest_dir} --plat manylinux_2_28_x86_64 {wheel}
+          CIBW_TEST_COMMAND: |
+            python -c "import os; print('SLEEF_PATH:', os.environ.get('SLEEF_PATH', 'Not set'))"
```

Comment:
Why test if `SLEEF_PATH` is set? That's just a detail of how you set up the build. Also if you delete this you can make the test command the same for all three architectures.